.nais/application/unleash-apitoken-prod.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: unleash.nais.io/v1
+kind: ApiToken
+metadata:
+  name: veilarbportefolje
+  namespace: pto
+  cluster: prod-fss
+  labels:
+    team: pto
+spec:
+  unleashInstance:
+    apiVersion: unleash.nais.io/v1
+    kind: RemoteUnleash
+    name: team-obo
+  secretName: veilarbportefolje-unleash-api-token
+
+  environment: production
```

Comment:
Disse filene trengs for at appen vet hvordan den skal generere api-token for unleash